services/movie.py

```diff
@@ -1,13 +1,15 @@
+from typing import Optional
+
 from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
-    title: str = None
+    genres_ids: Optional[list[int]] = None,
+    actors_ids: Optional[list[int]] = None,
+    title: Optional[str] = None
```

Comment:
it's better to use | operator instead of Optional as a moder way to specify not required parameters

Reply:
Fix everywhere